tests/framework/installer/ceph_manifests_previous.go

```diff
@@ -37,8 +42,16 @@ func (m *CephManifestsPreviousVersion) Settings() *TestCephSettings {
 	return m.settings
 }
 
-func (m *CephManifestsPreviousVersion) GetCRDs(k8shelper *utils.K8sHelper) string {
-	return m.settings.readManifestFromGitHub("crds.yaml")
+func (m *CephManifestsPreviousVersion) GetCRDs(t *testing.T, k8shelper *utils.K8sHelper) string {
+	k8sVersion, err := k8sutil.GetK8SVersion(k8shelper.Clientset)
+	if err != nil {
+		assert.NoError(t, err)
+	}
+	minValidatinPolicy := k8sVersion.AtLeast(version.MustParseSemantic(MinValidatinPolicy))
```

Comment:
typo, and seems like you can make it lower case if no other package uses it
```suggestion
	minValidatinPolicy := k8sVersion.AtLeast(version.MustParseSemantic(minValidationPolicy))
```